src/snapshot/tosvg.js

```diff
@@ -164,6 +164,8 @@ module.exports = function toSVG(gd, format, scale) {
     // Fix quotations around font strings and gradient URLs
     s = s.replace(DUMMY_REGEX, '\'');
 
+    // Do we need this process now that IE9 and IE10 are not supported?
```

Comment:
I don't know for sure but I'd guess we still need this. You're welcome to try and test but I'm not sure it's worth much time.